src/main/java/seedu/finance/logic/commands/SortCommand.java

```diff
@@ -22,13 +22,13 @@
     public static final String COMMAND_WORD = "sort";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the records in the list "
-            + "either by name, amount, date or category.\n"
+            + "either by name, amount, date or category based on the selected flag.\n"
             + "Parameters: FLAG \n"
-            + "Possible flags: "
-            + COMMAND_FLAG_NAME + ", "
-            + COMMAND_FLAG_AMOUNT + ", "
-            + COMMAND_FLAG_DATE + ", "
-            + COMMAND_FLAG_CATEGORY + "\n"
+            + "Flags: \n"
+            + COMMAND_FLAG_NAME + ": Sort records by name in lexicographical order\n"
+            + COMMAND_FLAG_AMOUNT + ": Sort records by amount in descending order\n"
+            + COMMAND_FLAG_DATE + ": Sort records by date from most recent to least recent\n"
+            + COMMAND_FLAG_CATEGORY + ": Sort records by category in lexicographical order\n"
```

Comment:
Can also include saying that if they want to sort:
- record by name in  reverse lexicographical order
- record by amount in ascending order
- record by date from earliest to latest record
- record by category in reverse lexicographical order

they can use the command `reverse`

Reply:
okay!